src/services/modules/rest/api/src/main/java/org/geoserver/geofence/services/rest/RESTAdminRuleService.java

```diff
@@ -40,6 +41,11 @@
     @Produces(MediaType.APPLICATION_XML)
     Response insert(@Multipart("rule") RESTInputAdminRule rule) throws BadRequestRestEx, NotFoundRestEx;
 
+    @POST
+    @Path("/all")
+    @Produces(MediaType.APPLICATION_XML)
+    Response insertAll(@Multipart("rule") List<RESTInputAdminRule> rules) throws BadRequestRestEx, NotFoundRestEx;
```

Comment:
Since we have a default implementation for the service I woull add it also here

Reply:
@taba90 The default implementation is for the DAO, and not the service.